indexing-service/src/main/java/io/druid/indexing/overlord/autoscaling/SimpleResourceManagementStrategy.java

```diff
@@ -48,15 +44,10 @@
 
 /**
  */
-public class SimpleResourceManagementStrategy implements ResourceManagementStrategy<RemoteTaskRunner>
+public class SimpleResourceManagementStrategy extends AbstractResourceManagementStrategy
```

Comment:
suggest changing AbstractResourceManagementStrategy to AbstractWorkerResourceManagementStrategy